src/controller/CHIPDeviceController.cpp

```diff
@@ -1490,18 +1490,22 @@ void DeviceCommissioner::OnUserDirectedCommissioningRequest(UDCClientState state
 
     ChipLogDetail(Controller, "------To Accept Enter: discover udc-commission <pincode> <udc-client-index>");
 }
+#endif // CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY
+
+#if CHIP_DEVICE_CONFIG_ENABLE_DNSSD
```

Comment:
When is this ever false?

We should remove this ifdef shouldn't we?

Reply:
I'm not sure it is ever false, but that's also a bit orthogonal to this PR.